modules/virt-vm-custom-scheduler.adoc

```diff
@@ -38,8 +38,9 @@ spec:
 
 
 .Verification
-* Verify that the VM is using the custom scheduler specified in the `VirtualMachine` manifest by checking the `virt-launcher` pod events:
-.. View the list of pods in your cluster by entering the following command:
+Verify that the VM is using the custom scheduler specified in the `VirtualMachine` manifest by checking the `virt-launcher` pod events:
+
+* View the list of pods in your cluster by entering the following command:
```

Comment:
I think the original formatting made more sense than the revised formatting does, because the second step is dependent on the first step (you get the name of the pod in the first step). I think it might need a slight phrasing update to make that clearer, though.

Reply:
I think I changed it because you don't need to do both steps to verify. Either step is sufficient. However, I set it back.